t/resty/signals.t

```diff
@@ -0,0 +1,184 @@
+# vi:ft= et ts=4 sw=4
+
+use lib 't/lib';
+use Test::Resty;
+
+plan tests => blocks() * 3;
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: Forward SIGINT to child process
+--- opts: -e 'print(1)'
+--- perlsrc
```

Comment:
I'm afraid the `perlsrc` section name is confusing here. I think you are just mocking an nginx here so maybe `--- mocked_nginx` is clearer here? It does not have to be a Perl script; it could be anything, like a shell script.